inc/functions/options.php

```diff
@@ -97,3 +97,20 @@ function imagify_load_network_options( $option_names, $prefixes = '' ) {
 
 	wp_cache_set( $notoptions_key, $notoptions, $cache_group );
 }
+
+/**
+ * Get admin stats.
+ *
+ * @param string  $key Option key.
+ * @param integer $default default value to return.
+ * @return mixed
+ */
+function imagify_get_admin_stats( string $key, $default = 0 ) {
```

Comment:
In relation to the option name change suggested, might be better to use `imagify_get_attachments_stats` or `imagify_get_media_stats`